src/main/java/sdmExam/Game.java

```diff
@@ -12,23 +12,22 @@ public Game() {
 
     protected static Game buildTestGame(Board board) { return new Game(board); }
 
-    public void play(Stone player, Position position) throws Exception {
+    public void play(Stone player, Position position) throws InvalidFirstPlayerException, RepeatedPlayException, OccupiedPositionException,IllegalMoveException {
```

Comment:
leave `throws Exception`, then we can capture the correct instance in the caller method that will use play.

Reply:
okay, less verbose more clear.

Reply:
done.